docs/src/operations/disk-space.md

```diff
@@ -0,0 +1,15 @@
+# Disk space
+
+Zeebe uses the local disk for storage of it's persistent data. Therefore if the Zeebe broker runs out of disk space the system is in an invalid state, as the broker cannot
+update it's state.
+
+To prevent the system to end in an unrecoverable state Zeebe expects a minimum size of free disk space available, and if this limit violated the broker will reject new
```

Comment:
```suggestion
To prevent the system to end in an unrecoverable state Zeebe expects a minimum size of free disk space available.
If this limit violated the broker will reject new
```